### PR TITLE
Modified Version Tests to pull request.

### PR DESCRIPTION
**Pull request by rasswanth-s**
## Description
Modified Syft VersionTests to pull requests, so we could run it manually in each PR , by modifying the file in ``packages/syft/tests/trigger/version_tests`` and fixes path filter issue.

## How has this been tested?
CI

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: .github/workflows/syft-security.yml [hunk 1]
```diff
@@ -60,13 +60,8 @@ jobs:
       - name: Install Dependencies
         if: steps.changes.outputs.syft == 'true'
         run: |
-          pip install packaging --default-timeout=60
```
Comment (madhavajay) [inline .github/workflows/syft-security.yml:63]: Unfortunately we need to install all the packages or the safety check doesn't work.

```
$ safety check
safety report
checked 24 packages, using free DB (updated once a month)
---
-> pillow, installed 8.2.0, affected <=8.2.0, id 40965

$ echo $?
255
 
$ pip uninstall pillow
Found existing installation: Pillow 8.2.0
Uninstalling Pillow-8.2.0
  Successfully uninstalled Pillow-8.2.0

$ safety check
safety report
checked 23 packages, using free DB (updated once a month)
---
No known security vulnerabilities found.

```

File: .github/workflows/syft-version_tests.yml [hunk 2]
```diff
@@ -9,9 +9,11 @@ name: Syft Version Tests
 on:
   schedule:
     - cron: "0 22 * * *" # 7pm Brazil, 10pm UTC, 8am AEST
-  push:
```
Comment (madhavajay) [inline .github/workflows/syft-version_tests.yml:12]: awesome! :)

**Comment by madhavajay** [comment]
@rasswanth-s Awesome work, if you can revert the package install for the security check step then we can merge! :)

File: .github/workflows/syft-security.yml (hunk 1 above)
Comment (rasswanth-s) [reply .github/workflows/syft-security.yml]: Reverted Back

**Review by madhavajay — APPROVED** [review]
Awesome work @rasswanth-s!

**Pull request merged**